packages/libnfs.rb

```diff
@@ -1,40 +1,27 @@
-# Adapted from Arch Linux libnfs PKGBUILD at:
-# https://github.com/archlinux/svntogit-community/raw/packages/libnfs/trunk/PKGBUILD
-
```

Comment:
Why was this note removed?  It might be helpful to keep this.

Reply:
because i rewrote the file so much that it no longer resembles that PKGBUILD file.